code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -241,14 +241,16 @@
 	return ..()
 
 /mob/living/carbon/xenomorph/pull_response(mob/puller)
+	if(stat != CONSCIOUS) // If the Xeno is unconscious, don't fight back against a grab/pull
+		return TRUE
+	if(!ishuman(puller))
+		return TRUE
 	var/mob/living/carbon/human/H = puller
-	if(stat == CONSCIOUS && H.species?.count_human) // If the Xeno is conscious, fight back against a grab/pull
```

Comment:
is count human used anywhere?

Reply:
checked with headmins, found references to it for `medholocard` and `/datum/ammo/bullet/shotgun/beanbag/on_hit_mob()`